src/RubberDuckyParking/templates/base.html

```diff
@@ -28,7 +28,7 @@
                     </li>
                     <li class="nav-item">
                         {% if user.is_authenticated %}
-                        <a class="nav-link" href="/user/123/">{{user.username}}</a>
+                        <a class="nav-link" href="/user/">{{user.username}}</a>
                         <a class="nav-link" href="{% url 'logout' %}"> Log out </a>
```

Comment:
How does the 'url' keyword work? what global variable does this link to and how can we alter it?